src/services/navigationBar.ts

```diff
@@ -700,4 +703,16 @@ namespace ts.NavigationBar {
                 return false;
         }
     }
+
+    function cleanText(text: string): string {
+        // Truncate to maximum amount of characters as we don't want to do a big replace operation.
+        text = text.length > maxLength ? text.substring(0, maxLength) + "..." : text;
+
+        // Replaces ECMAScript line terminators and removes the trailing `\` from each line:
+        // \n - Line Feed
+        // \r - Carriage Return
+        // \u2028 - Line separator
+        // \u2029 - Paragraph separator
+        return text.replace(/\\?(\r?\n|\u2028|\u2029)/g, '');

```

Comment:
What are they replaced with?  I thought we were going to use the newline symbol character.

Reply:
I decided it was better to just remove the line terminators. I believe this is a more correct way of representing the token on the navbar. 

Check the scenarios below:
```js
const x = { "a\
b": 1};
```
is the same as:
```js
const x = { "ab": 1 };
```
navbar will display `ab` for both of them.